### DISCUSSION TITLE
Fresh new version 2.0 beta announcement

### DISCUSSION THREAD

**Discussion by alexhua** [alexhua/Aria2-for-chrome#118] [Announcements]
We have released a fresh new version 2.0 beta on the [mv3](https://github.com/alexhua/Aria2-for-chrome/tree/mv3) branch.

The main motivation behind this update is to migrate the extension from Manifest V2 to Manifest V3, in line with Google Chrome's extension planning.

The extension code has been fully re-written and the key changes are listed below:
1. Made a new name `Aria2 Explorer` for extension
2. Changed the open source license to `BSD 3-Clause`
3. Combine Chrome and Edge version into all-in-one package
4. Adapted the extension to meet the requirements of Manifest V3
5. Refurbished the extension options page
6. Exported all RPC settings to AriaNG when they are created or changed
7. Added an option to receive task complete or error notifications when `Monitor Aria2` is enabled
8. Added an option to keep the system awake when 'Monitor Aria2' is enabled
9. Many other detailed improvement

It's a fully new version. 

Test method:
1. Download the extension Aria2E-v2.0-beta.zip
2. Extract it to a folder
3. Go to browser extension page `chrome://extensions/` or `edge://extensions`
4. Enable developer mode, load the folder created in step 2

We would appreciate more enthusiastic users to participate in testing and provide feedback. 

<hr /><em>This discussion was created from the release <a href='https://github.com/alexhua/Aria2-for-chrome/releases/tag/v2.0-beta'>Fresh new version 2.0 beta announcement</a>.</em>

**Comment by MbeleGarland** [comment]
OK I realize this was closer to the beginning of the year but Ill take ya up on the feedback,. Not to sure about that whole enthusiasm though ;-)
